client/src/components/GraduateForm.js

```diff
@@ -114,6 +129,11 @@ const GraduateForm = ({ profile, handleClick }) => {
                 name="surname"
                 label="Last Name"
               />
+              <FormField
+                name="email"
+                description="if you don't want your e-mail address to be public, please add contact@codeyourfuture.com instead"
```

Comment:
I know we said to put a message like this on Saturday but I think it's better user experience if they don't have to.
I would just make it clear that it's optional to provide email address and then I'd conditionally put "contact@codeyourfuture.com" in the request body if the user didn't provide a different email address.